### PR TITLE
New editor as default

### PR DESCRIPTION
**Pull request by ericnewcomer**
* Update to flow editor v1.6.10 which includes more ordinals on sms form
 * Makes new editor the default and removes beta flags (but doesn't remove option to create flows with old editor)

### CODE REVIEW

**Comment by ericnewcomer** [comment]
What do you mean what do they mean?
https://github.com/nyaruka/floweditor/blob/master/CHANGELOG.md

Or are you referring to the broken attempt at semver with some of these changes?

**Comment by nicpottier** [comment]
No no, was joking about the `0` vs `1` and use of version in the Django forms. We have so many ways of representing versions for engine / editor etc..

**Comment by ericnewcomer** [comment]
Ha, ya, it's a little silly for sure, probably not the best name for that. But ya, gone soon.

File: temba/flows/views.py [hunk 1]
```diff
@@ -396,10 +396,9 @@ class FlowCreateForm(BaseFlowForm):
                 ),
             )
 
-            use_new_editor = forms.TypedChoiceField(
-                label=_("New Editor (Beta)"),
-                help_text=_("Use new editor when authoring this flow"),
-                choices=((1, "Yes"), (0, "No")),
+            editor_version = forms.TypedChoiceField(
+                help_text=_("If you are unsure, use the new editor"),
+                choices=((0, "New Editor"), (1, "Previous Editor")),
```
Comment (rowanseymour) [inline temba/flows/views.py]: these should be `_()` strings

Comment (ericnewcomer) [reply temba/flows/views.py]: But should they really? They are both going away pretty quickly. I'm okay with them not being localized.

**Pull request merged**